components/contributors/ContributorsList.js

```diff
@@ -28,7 +28,17 @@ const ContributorsList = ({ contributors, isFounders }) => {
 }
 
 ContributorsList.propTypes = {
-  contributors: PropTypes.array.isRequired,
+  contributors: PropTypes.arrayOf(
```

Comment:
The original list is all the properties in the object, but we never use them all, so refining the prop types for a contributor to the properties we render.